packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -170,6 +177,7 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
         ResourceMetering.ResourceConfig memory _config,
         uint256 _startBlock,
         address _batchInbox,
+        address _challenger,
```

Comment:
Stack too deep here? If so, we should consolidate the gas config into a struct that is read into memory

Reply:
A type that has the overhead, scalar, gaslimit together seems like a good type to combine into gas config